src/main/java/soot/toDex/StmtVisitor.java

```diff
@@ -430,6 +431,7 @@ public void caseAssignStmt(AssignStmt stmt) {
 		}
 		Local lhsLocal = (Local) lhs;
 
+		final Insn newStmt;
```

Comment:
Bitte newInsn nennen. Wir sollten in der Nomenklatur strikt trennen, was ein Statement (Soot) und was eine Instruction (Dex) ist.